packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -537,7 +540,12 @@ const CFDPasswordForm = ({
                                         i18n_default_text='Enter your {{platform}} password to add a {{platform_name}} {{account}} {{jurisdiction_shortcode}} account.'
                                         values={{
                                             platform: getCFDPlatformLabel(platform),
-                                            platform_name: platform === CFD_PLATFORMS.MT5 ? 'MT5' : 'Deriv X',
+                                            platform_name:
+                                                platform === CFD_PLATFORMS.MT5
+                                                    ? 'MT5'
+                                                    : platform === CFD_PLATFORMS.DXTRADE
+                                                    ? 'Deriv X'
+                                                    : 'cTrader',
```

Comment:
it not a good practice to use multiple ternary operator.
instead make a variable and put the else condition in that variable.
Eslint will giver errors on this maybe
`const is_ctrader = platform === CFD_PLATFORMS.DXTRADE ? 'Deriv X' : 'cTrader'`
```suggestion
                                            platform_name:
                                                platform === CFD_PLATFORMS.MT5
                                                    ? 'MT5'
                                                    : is_ctrader,
```